transports/msmq/scaling-out_overview_core_6.partial.md

```diff
@@ -14,8 +14,9 @@ Following table compares both approaches:
 | No [flow control](https://en.wikipedia.org/wiki/Flow_control_%28data%29)| [Flow control](https://en.wikipedia.org/wiki/Flow_control_%28data%29) via ready messages             |
 | No automatic worker failure detection       | Worker failure detection via ready messages |
 | Round-robin load balancing                  | Fair load balancing                         |
+| Available since core v6 & msmq v1           | Unsupported since core v7 / msmq v1         |
```

Comment:
There should be a separate partial for V7 that says that SSD is the only option supported. For V7 users there is no need to read this table only to learn that the SSD is their only option.